app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/GoogleSignInWidget.kt

```diff
@@ -94,16 +137,50 @@ class GoogleSignInWidget : Fragment() {
         }
     }
 
+    private fun setSingInUi() {
+        view?.findViewById<Button>(R.id.singOutButton)?.visibility = View.GONE
+        view?.findViewById<Button>(R.id.checkBox)?.visibility = View.VISIBLE
+        view?.findViewById<Button>(R.id.loginButton)?.visibility = View.VISIBLE
+    }
+
+    private fun setSingOutUi() {
```

Comment:
I forgot this one haha,  I am sure that you're a good singer but I think sign is better here. ;)